apps/jobs/urls.py

```diff
@@ -110,6 +110,12 @@
         views.update_submission_started_at,
         name="update_submission_started_at",
     ),
+    url(
+        r"^challenge/(?P<challenge_pk>[0-9]+)/"
```

Comment:
`challenge/(?P<challenge_pk>[0-9]+)/"` -> `challenges/(?P<challenge_pk>[0-9]+)/"`

Reply:
@Ram81 I think you missed this comment.